util/marketplace.py

```diff
@@ -102,7 +105,13 @@ def lookup_subscription(self, ebsAccountNumber, skuId):
         except json.decoder.JSONDecodeError:
             return None
 
-        return subscription
+        if subscription:
+            end_date = subscription["effectiveEndDate"]
+            now = time.time() * MILLISECONDS
```

Comment:
nit: instead of MILLISECONDS, i would indicate it in the variable's name: `now_ms = time.time() * 1000`.
Also, do you need to round the value.

Reply:
"effectiveEndDate" is in milliseconds, which is why I rounded it here.